test/parallel/begin/deitz/test_begin.chpl

```diff
@@ -3,12 +3,15 @@ use Time;
 const n = 5;
 
 var a: [1..n] int;
+var B: [1..n] sync bool;
```

Comment:
By convention (according to the spec) sync and single vars should end with '$' as an indication that references to them may block. So maybe rename to B$?